docs/onboarding/01-project-overview.md

```diff
@@ -0,0 +1,92 @@
+# Yellowstone Vixen - Project Overview
+
+## Introduction
+Yellowstone Vixen is a high-performance Rust-based project that provides a gRPC service for parsing Solana transactions. It's designed to work seamlessly with the Yellowstone ecosystem, offering efficient and reliable transaction parsing capabilities.
+
+## Architecture Overview
+
+```mermaid
+graph TB
+    Client[Client Applications] -->|gRPC Requests| Vixen[Yellowstone Vixen Service]
+    Vixen -->|Parse| Transactions[Solana Transactions]
+    Vixen -->|Metrics| Prometheus[Prometheus Monitoring]
+    Vixen -->|Tracing| OpenTelemetry[OpenTelemetry]
+    
+    subgraph "Vixen Service Components"
+        Parser[Transaction Parser]
+        ProtoGen[Protocol Buffer Definitions]
+        Metrics[Metrics Collection]
+        Config[Configuration Management]
+    end
+
+    Vixen --> Parser
+    Vixen --> ProtoGen
+    Vixen --> Metrics
+    Vixen --> Config
+```
+
+## Key Components
+
+### 1. gRPC Service
+- Provides a high-performance interface for transaction parsing
+- Uses Protocol Buffers for efficient data serialization
+- Supports bi-directional streaming capabilities
+- Configurable connection pooling and resource management
```

Comment:
Vixen starts with a raw change event stream but can be ran in a mode that produces a grpc stream. This seems to be mixing that. I think this should be about the input grpc stream and document vixen streams as a available mode or extension.

This docs should be about this service and what vixen does 
https://github.com/rpcpool/yellowstone-grpc

- Generates GRPC request based on registered parsers. It will subscribe to programs that are registered to parsers.
- Create subscription and handles the stream

Reply:
Gotcha, thanks for sharing! I'm creating these info of this project in order to understand this and helping other developers as well